tests/constants.py

```diff
@@ -6,3 +6,9 @@
 INPUTS_COPY = "inputs"
 
 REPO_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+INPUT_FOLDER = "inputs"
```

Comment:
I would use the values from `src/constants.py`

Reply:
Okay, I'll use them, but for values that are different (I will have that in the tests for D1, also the repo path is different) we should keep the `constants.py` in 'tests` folder. Do you agree?

Reply:
```suggestion
from src.constants import INPUT_FOLDER
```

Reply:
Yes I agree that the variable `JSON_PATH` and `REPO_PATH` should be defined in the `tests/constants.py`, only `INPUT_FOLDER` and `JSON_FNAME` are to be imported from `src/constants.py`